addon/components/code-base.js

```diff
@@ -0,0 +1,20 @@
+/* global Prism */
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+
+export default Component.extend({
+  classNameBindings: ['languageClass'],
+  language: 'markup',
+
+  languageClass: computed('language', function() {
+    return 'language-' + this.get('language');
+  }),
+
+  getElement() {
+    return this.element;
+  },
+
+  didInsertElement() {
+    Prism.highlightElement(this.getElement());
```

Comment:
do you need to do anything to remove the bindings on `willDestroyElement`?

If the bindings stick around in some way it could cause a performance problem eventually

Reply:
We weren't doing anything before, but I will look into it.

Reply:
not even sure it's a problem, but thought I'd bring it up